be1-go/README.md

```diff
@@ -2,109 +2,27 @@
-                                                                        `signature` both encoded in base 64 because we receive them encoded.
-* For the signature, we exclusively use the unencoded representation. 
-
-### test
-This package contains files useful for manual real-world testing.
-
-### config
-This package contains configurations files for constants. The file starting with `// +build [param]` will be used when using the command `go build -tags [param]`. The default file should start with `// +build ![param1],![param2],![param3]`.
-
-## Back-end components
 ### Dependencies
 Currently, the project works with the following libraries:
 * [gorilla/websocket](https://github.com/gorilla/websocket) is a websocket package for golang. We chose to use this 
 package because it offers a good API, and it is more complete than the websocket package offered by the standard library.
-* [boltdb](https://github.com/boltdb/bolt) is a low-level key-value DB management system. It works by creating Buckets, 
-that we can populate with key-value pairs.
-
-### Database structure
-When running a back-end server, the program will check if the database specified as argument already exists. If it does 
-it will open it, and will create it otherwise. The database will have the exact same structure whether you run the server 
-as an organizer or as a witness. The contents may differ, but the structure stays the same.
-
-The database is going to store 2 types of bucket:
-* The first type is called `channels`, and it is a unique bucket. For each channel, it contains a (key, value) pair where the key is the channel's ID
-and the value is a string containing the current state of the channel. There is no history of messages stored there, 
-only the channel's name, organizer, list of witnesses, etc...
-* The second type of bucket are buckets containing the history of one channel. There is one such bucket per channel, and 
-it has the following properties :
-    1. The bucket name is the ID of the channel
-    2. The (key, values) pairs are (ID, content) of messages sent on the channel. The content is the field `message` that
-  is contained in the field `params` of the received json message.
 
-Once a message is stored, it shall not be updated, except for messages requesting witness signing. In that case, we 
-append the received witness signatures at the end of the original message's list of signatures.
 
 ## Unit-tests
 Some unit-tests exist directly in their corresponding packages, as per Go specification. They can be run with `go test -v ./...` (recursive from the top-level folder) or simply `go test` for the current package.
 
-## Basic testing
-We provide a very basic front-end in the file `test/index.html`. It does nothing but opens a websocket connection to the back-end server, and lets
-you send and receive messages. IT's accessible from the `address`:`port`/test URL. There you can the JSON-RPC requests examples given in `test/json_test_strings.txt`
```

Comment:
isn't this still relevant?

Reply:
I feel the effort to implement this has been replicated across all projects. Something https://hoppscotch.io would work just fine to test things out. Agreed the test_strings can be handy though. Adding them back